go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -715,6 +720,10 @@ func transformToDoltRow(row row.Row, rdSchema schema.Schema, wrSchema sql.Schema
 	for i, col := range wrSchema {
 		switch col.Type {
 		case sql.Boolean, sql.Int8, sql.MustCreateBitType(1): // TODO: noms bool wraps MustCreateBitType
+			switch doltRow[i].(type) {
+			case int8:
+				doltRow[i] = strconv.Itoa(int(doltRow[i].(int8)))
```

Comment:
Have two cases. One from int8 and the other for string. No need to do an additional string conversion. The int8 can be left alone. The boolean type in the sqlengine should process it accordingly.